app/controllers/assignments_controller.rb

```diff
@@ -1,102 +1,102 @@
 class AssignmentsController < ApplicationController
-	before_filter :userLoggedIn
-	before_filter :courseExists, :only => [:new, :create]
-	before_filter :assignmentExists, :only => [:show, :edit, :update, :delete]
+  before_filter :userLoggedIn
+  before_filter :courseExists, :only => [:new, :create]
+  before_filter :assignmentExists, :only => [:show, :edit, :update, :delete]
   before_filter :partOfCourse, :only => [:show]
-	before_filter :authCourseEdit, :except => [:show]
+  before_filter :authCourseEdit, :except => [:show]
 
-	def show
-		@assignment = Assignment.find_by(id: params[:id])
-	end
+  def show
+    @assignment = Assignment.find_by(id: params[:id])
+  end
 
-	def new
-		#render the page to create new assignment
+  def new
+    # render the page to create new assignment
     @course = Course.find(params[:course_id])
-	end
+  end
 
-	def create
-		@assignment = Assignment.new(assignment_params)
-		if @assignment.valid?
-			@course.assignments << @assignment
+  def create
+    @assignment = Assignment.new(assignment_params)
+    if @assignment.valid?
+      @course.assignments << @assignment
       flash[:message] = "You have created this assignment"
       redirect_to assignment_show_path(@assignment) and return
     else
-    	render 'new'
-    	return
-		end
-	end
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.